jujugui/static/gui/src/app/components/mid-point/_mid-point.scss

```diff
@@ -0,0 +1,44 @@
+.mid-point {
+    color: $cool-grey;
+    padding: 30px 20px;
+
+    &__title {
+        margin-bottom: 20px;
+        font-size: 22px;
+    }
+
+    &__charm-list {
+        display: flex;
+        // The list items have margin right and we don't want a double gap on
+        // the right.
+        margin: 0 -20px 0 0;
```

Comment:
This would be better combated with `.mid-point__charm:last-of-type { margin-right: 0 }` instead of using -margins as they can be confusing.